### PR TITLE
[Obmin-299] replacing the import of icons

### PR DESCRIPTION
**Pull request by AkintsevaElizaveta**

### CODE REVIEW

File: src/main/resources/react/src/components/common/buttons/btnGoodBusiness/index.jsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Link } from 'react-router-dom';
-import styled from 'styled-components';
+import { Icon } from '@wolshebnik/obminyashka-components';
 
-import { ReactComponent as HeartSvg } from 'assets/icons/heart.svg';
+import styled from 'styled-components';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/buttons/btnGoodBusiness/index.jsx:4]: Remove space and sort

File: src/main/resources/react/src/components/item-card/styles.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable indent */
-import styled, { css } from 'styled-components';
+import { Icon } from '@wolshebnik/obminyashka-components';
 
-import * as Icon from 'assets/icons';
+import styled, { css } from 'styled-components';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/item-card/styles.js:4]: Remove space and sort

File: src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto/styles.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { Icon } from '@wolshebnik/obminyashka-components';
+
 import styled, { css } from 'styled-components';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/ProductPage/ProductPhotoCarousel/NoPhoto/styles.js:3]: remove space and sort

File: src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/calendar/header/styles.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import styled, { css } from 'styled-components';
+import { Icon } from '@wolshebnik/obminyashka-components';
 
-import * as Icon from 'assets/icons';
+import styled, { css } from 'styled-components';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tab_pages/my_profile/children/calendar/header/styles.js:3]: remove space and sort

File: src/main/resources/react/src/pages/UserInfo/tabs/config.js [hunk 5]
```diff
@@ -1,32 +1,32 @@
 import { route } from 'routes/routeConstants';
 
-import * as Icon from 'assets/icons';
+import { Icon } from '@wolshebnik/obminyashka-components';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/UserInfo/tabs/config.js:3]: lib should be first

**Review by Voloshkina — APPROVED** [review]

**Review by Viktoria-Serkiz — APPROVED** [review]

**Review by Kyslytsia — APPROVED** [review]

**Review by ASGladchenko — APPROVED** [review]

**Review by Mish-hash — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**